test/plugin/test_out_bigquery.rb

```diff
@@ -261,69 +255,64 @@ def test_configure_invalid_fieldname
+  def test_format
     now = Time.now
-    input = [
-      now,
-      {
-        "status" => "1",
-        "bytes" => 3.0,
-        "vhost" => :bar,
-        "path" => "/path/to/baz",
-        "method" => "GET",
-        "protocol" => "HTTP/0.9",
-        "agent" => "libwww",
-        "referer" => "http://referer.example",
-        "requesttime" => (now - 1).to_f.to_s,
-        "bot_access" => true,
-        "loginsession" => false,
-        "something-else" => "would be ignored",
-        "yet-another" => {
-          "foo" => "bar",
-          "baz" => 1,
-        },
-        "remote" => {
-          "host" => "remote.example",
-          "ip" =>  "192.0.2.1",
-          "port" => 12345,
-          "user" => "tagomoris",
-        }
+    input = {
+      "status" => "1",
+      "bytes" => 3.0,
+      "vhost" => :bar,
+      "path" => "/path/to/baz",
+      "method" => "GET",
+      "protocol" => "HTTP/0.9",
+      "agent" => "libwww",
+      "referer" => "http://referer.example",
+      "requesttime" => (now - 1).to_f.to_s,
+      "bot_access" => true,
+      "loginsession" => false,
+      "something-else" => "would be ignored",
+      "yet-another" => {
+        "foo" => "bar",
+        "baz" => 1,
+      },
+      "remote" => {
+        "host" => "remote.example",
+        "ip" =>  "192.0.2.1",
+        "port" => 12345,
+        "user" => "tagomoris",
       }
-    ]
+    }
     expected = {
-      "json" => {
-        "time" => now.to_i,
-        "status" => 1,
-        "bytes" => 3,
-        "vhost" => "bar",
-        "path" => "/path/to/baz",
-        "method" => "GET",
-        "protocol" => "HTTP/0.9",
-        "agent" => "libwww",
-        "referer" => "http://referer.example",
-        "requesttime" => (now - 1).to_f.to_s.to_f,
-        "bot_access" => true,
-        "loginsession" => false,
-        "something-else" => "would be ignored",
-        "yet-another" => {
-          "foo" => "bar",
-          "baz" => 1,
-        },
-        "remote" => {
-          "host" => "remote.example",
-          "ip" =>  "192.0.2.1",
-          "port" => 12345,
-          "user" => "tagomoris",
-        }
+      "time" => now.to_i,
+      "status" => 1,
+      "bytes" => 3,
+      "vhost" => "bar",
+      "path" => "/path/to/baz",
+      "method" => "GET",
+      "protocol" => "HTTP/0.9",
+      "agent" => "libwww",
+      "referer" => "http://referer.example",
+      "requesttime" => (now - 1).to_f.to_s.to_f,
+      "bot_access" => true,
+      "loginsession" => false,
+      "something-else" => "would be ignored",
+      "yet-another" => {
+        "foo" => "bar",
+        "baz" => 1,
+      },
+      "remote" => {
+        "host" => "remote.example",
+        "ip" =>  "192.0.2.1",
+        "port" => 12345,
+        "user" => "tagomoris",
       }
     }
 
     driver = create_driver(CONFIG)
-    driver.instance.start
-    buf = driver.instance.format_stream("my.tag", [input])
-    driver.instance.shutdown
+    driver.instance_start
```

Comment:
Or, output plugin test driver in v0.14 remembers all return values of `#format` method and it can be taken by `driver.formatted`. It's also available to test `#format`.